blaze/server/client.py

```diff
@@ -130,8 +143,9 @@ def compute_down(expr, ec, **kwargs):
     serial = ec.serial
     r = post(
         ec,
-        'compute.{serial}'.format(serial=serial.name),
+        '/compute',
```

Comment:
this will generate 2 slashes. we might want to update the `_request` function to not add the slash.